framework/test/unit/modules/interoperability/mainchain/commands/recover_message.spec.ts

```diff
@@ -59,7 +59,13 @@ import {
 import { CcmSendSuccessEvent } from '../../../../../../src/modules/interoperability/events/ccm_send_success';
 import { InvalidRMTVerification } from '../../../../../../src/modules/interoperability/events/invalid_rmt_verification';
 
-const getHeight = (size: number) => Math.ceil(Math.log2(size)) + 1;
+// Choose the index such that the position of the ccm in the outbox tree is at terminatedChainOutboxSize,
+// i.e. just outside of the tree. Note that a 1 has to be prepended to the binary representation of the index, which is done by adding 2**indexLength.
+// See https://github.com/LiskHQ/lips/blob/main/proposals/lip-0031.md#proof-serialization.
+const appendPrecedingToIndices = (indices: number[], terminatedChainOutboxSize: number) => {
```

Comment:
We can either move this inside the first level describe, or move the two createContext functions where this is, to keep it consistent

Reply:
Updated